app/src/main/java/at/tomtasche/reader/background/PdfLoader.java

```diff
@@ -47,7 +47,7 @@ public void loadSync(Options options) {
             File cacheDirectory = AndroidFileCache.getCacheDirectory(cacheFile);
 
             pdf2htmlEX pdfConverter = new pdf2htmlEX(context).setInputPDF(cacheFile);
-            pdfConverter.setOutline(false).setBackgroundFormat("jpg").setDRM(false);
+            pdfConverter.setDRM(false);
```

Comment:
@TomTasche @ViliusSutkus89 do you know what should be done here? It complained about the functions being not existent and I just removed the calls for now.

Reply:
if I remember correctly that disables the overview of pages on the left side, so it was quite important for us! can you remember why it was removed? @ViliusSutkus89

Reply:
Maybe it was just replaced with a different method?

What confused me is why does this break now? Are we not pinning the version correctly of this library?

Reply:
the outline is now visible (black bar on the left in screenshot), which is not what we want. we can't release like that :disappointed: help @ViliusSutkus89 
![image](https://github.com/opendocument-app/OpenDocument.droid/assets/128734/5d4c03dd-4554-4c12-ada0-304a676e4924)

Reply:
Hey, is this problem still relevant? I've tried running current main version and outline isn't there

Reply:
I actually didn't try and just made it compile. Did you try @TomTasche ?

Reply:
I did, all good!